core/common/src/test/java/alluxio/collections/LockCacheTest.java

```diff
@@ -82,4 +83,30 @@ public void parallelInsertTest() throws InterruptedException {
     t3.join();
     t4.join();
   }
+
+  @Ignore
+  @Test(timeout = 100000)
```

Comment:
can we tune the test to stay under 1 second so that we can include it in the build?

Reply:
forgot to remove this..